users/id-b3/pimoroni_trackball.c

```diff
@@ -0,0 +1,203 @@
+/*
```

Comment:
Just a heads up, this file exists in /drivers/sensors/, in develop, and this copy may be removed in the future.